### PR TITLE
fix: Update config and append influx docker config

### LINKED ISSUE #186: Bug: Update openkf config.
### What happened?

Some service may not be attached correctly.

### What did you expect to happen?

Fix config.

### How can we reproduce it (as minimally and precisely as possible)?

ALL.

### Anything else we need to know?

ALL.

### version

ALL.

### Cloud provider

ALL.

### OS version

ALL.

### Install tools

ALL.

### PR DESCRIPTION
**Pull request by IRONICBo**
#### 🔍 What type of PR is this?

/kind bug

#### 👀 What this PR does / why we need it:

Update service config.

#### 🅰 Which issue(s) this PR fixes:

Fixes #186

### CODE REVIEW

**Comment by IRONICBo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by IRONICBo** [comment]
recheck

File: deploy/docker-compose/docker-compose.yaml [hunk 1]
```diff
@@ -93,15 +72,34 @@ services:
...
+    environment:
+      - DOCKER_INFLUXDB_INIT_MODE=setup
+      - DOCKER_INFLUXDB_INIT_USERNAME=admin
+      - DOCKER_INFLUXDB_INIT_PASSWORD=admin1234
+      - DOCKER_INFLUXDB_INIT_ORG=openkf
+      - DOCKER_INFLUXDB_INIT_BUCKET=openkf
+      - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=admin1234
+    networks:
+      openkf-network:
+        ipv4_address: 177.8.0.6
```
Comment (charles-chenzz) [inline deploy/docker-compose/docker-compose.yaml:97]: where does this IP point to?

File: server/config.example.yaml [hunk 2]
```diff
@@ -61,11 +61,11 @@ email:
   pool_size: 4
 
 openim:
-  secret: openkf
+  secret: openIM123
   ip: 127.0.0.1
   api_port: 10002
   platform_id: 5 # web
-  admin_id: openkf_admin # OpenIMServer admin userID
+  admin_id: openIMAdmin
```
Comment (charles-chenzz) [inline server/config.example.yaml:68]: this is integration with the im part?

**Review by cubxxw — APPROVED** [review]

File: deploy/docker-compose/docker-compose.yaml (hunk 1 above)
Comment (IRONICBo) [reply deploy/docker-compose/docker-compose.yaml:97]: @charles-chenzz These IPs are used in cluster.

File: server/config.example.yaml (hunk 2 above)
Comment (IRONICBo) [reply server/config.example.yaml:68]: It's easy to use in default openim server config.

**Pull request merged**